### PR TITLE
[DOC-263] added note on setting gflags for live migration 

### PR DESCRIPTION
**Pull request by aishwarya24**
@netliffy /preview/yugabyte-voyager/migrate/live-migrate/#create-the-target-database

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md [hunk 1]
```diff
@@ -165,6 +165,25 @@ If you don't provide the target YugabyteDB database name during import, yb-voyag
 CREATE DATABASE target_db_name;
 ```
 
+{{<note title="Important">}}
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md:168]: Can we shift this note just before the "create the target database" section ? Like we should let user know this while preparing target cluster.

**Review by priyanshi-yb — APPROVED** [review]
LGTM! Thanks for this quick change!

File: docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md [hunk 2]
```diff
@@ -155,6 +155,25 @@ You can use only one of the following arguments to connect to your Oracle instan
 
 Prepare your target YugabyteDB database cluster by creating a database, and a user for your cluster.
 
+{{<note title="Important">}}
+
+Add the following flags to the cluster before starting migration, and revert them after the migration is complete:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md:160]:
```suggestion
Add the following flags to the cluster before starting migration, and revert them after the migration is complete.
```

**Review by ddhodge — APPROVED** [review]
LGTM except the colon

**Pull request merged**